corehq/apps/users/models_role.py

```diff
@@ -161,7 +161,12 @@ def to_json(self):
     def set_permissions(self, permission_infos):
         def _clear_query_cache():
             try:
-                self.refresh_from_db(fields=["rolepermission_set"])
+                # There is a bug in refresh_from_db when specifying fields that results in this error:
+                # RuntimeError: Set changed size during iteration
+                # Once on a version of Django that includes the change made for
```

Comment:
Are you keeping this on your radar, or how will we be reminded to do this?

Reply:
https://docs.google.com/spreadsheets/d/1t1LWSlvxzN7WwCFbFBmisM1p33PBIfbqXhm6vJClPeg/edit#gid=663153186